app/views/relationships/index.html.haml

```diff
@@ -0,0 +1,21 @@
+%section.people
+  %header
+    %h2 People I Follow
+  %table.table
+    %thead
+      %tr
+        %th(width="30%")
+        %th(width="20%") Videos in Queue
+        %th(width="20%") Followers
+        %th(width="30%") Unfollow
+    %tbody
+      - @relationships.each do |relationship|
+        %tr
+          %td
+            %img(src="http://www.gravatar.com/avatar/#{Digest::MD5.hexdigest('knwang@gmail.com'.downcase)}?s=40")
+            = link_to "#{relationship.followed_user.full_name}", user_path(relationship.followed_user)
+          %td.extra-padding #{relationship.followed_user.queue_items.count}
+          %td.extra-padding #{relationship.followed_user.followers.count}
+          %td.extra-padding
+            = link_to relationship_path(relationship), method: :delete do
```

Comment:
I would rather use shorthand route `relationship` instead of  `relationship_path(relationship)`.